frontend/resourceadm/pages/ErrorPage/ErrorPage.module.css

```diff
@@ -0,0 +1,4 @@
+.pageWrapper {
+  padding-inline: 10%;
+  padding-block: 50px;
+}
```

Comment:
Kjørte opp koden for å sjekke denne pagen nå. Ser ut som noe har skjedd med cssen? Kan du sjekke ut hva det er sånn at vi har en nav bar som er lik på alle sider. Dette er spesielt viktig nå som vi snart skal ha en demo, og et feiltrykk kan resultere i at vi ender på denne error siden. 
Sånn ser det ut i min browser:
<img width="1728" alt="Screenshot 2023-06-20 at 13 27 29" src="https://github.com/Altinn/altinn-studio/assets/133344438/759e9aa5-ad97-47f6-a129-b9a4f58f1419">

Reply:
Ja, jeg har sett den CSS feilen i VersionControlHeader før. Det skjer dersom content på siden blir for lite. Har ikke brukt noe særlig tid på det. Dyttet vel bare inn en kjapp fiks så det så bra ut med det opprinnelige oppsettet. Skal se om reversjon til h1-tags vil fikse det.

Reply:
Det kan veldig raskt fikses med å oppdatere cssen på ```pageWrapper``` i filen ```ErrorPage.module.css``` slik: ```min-height: 100vh;``` Da forcer du siden til å ha en minimum høyde.

Reply:
Da har jeg fikset problemet for error-sidene.

<img width="1145" alt="demo_errorPage_med_padding" src="https://github.com/Altinn/altinn-studio/assets/94476391/49b091c3-c499-4da3-8696-5c5b1ebb13ae">

Reply:
Men det er en feil i banner som etterhvert bør fikses. Slik ser det f.eks ut på ResourceDashboardPage om man tar vekk ressurs-listen og søkefeltet :
<img width="1132" alt="demo_banner2_cssBug" src="https://github.com/Altinn/altinn-studio/assets/94476391/86565e69-9757-4fea-9d28-59eaaeb4a580">

Reply:
@Torgeir333 det løser dessverre ikke problemet å legge på ```padding-bottom```. Om du tester det du har laget på en større skjerm vil du se at problemet vedvarer. Som nevnt over, bruk ```min-height: 100vh``` og problemet er løst. 
![Screenshot 2023-06-20 at 15 37 23](https://github.com/Altinn/altinn-studio/assets/133344438/6b4ece55-d1ee-4872-b406-7816a811beca)

Alternativt så kan du løse problemet som du nevner over once and for all med å legge til 1 linje med css. Jeg var ikke klar over at det var et issue, så det var veldig bra spottet av deg! 
Om du går inn i filen ```resourceadm/pages/PageLayout/VersionControlHeader/VersionControlHeader.module.css``` og legger på ```height: 50px``` i ```.wrapper``` så er problemet løst once and for all, og du slipper legge på ```min-height: 100vh``` i de andre komponentene. Da er også problemet løst som du nevnte for de andre sidene også :)

Reply:
Jeg prøvde 100vh (ca 1000 px), men det dyttet Footer ut av skjermen, derav 200 px, som fungerte på min Mac og på PC. Ja, det er nok noe som bør fikses i banner-komponenten, men det er nok out-of-scope i denne PR.

Reply:
Det er noe som påvirker pagen du prøver å merge inn, i tillegg til at det er en såpass liten fix som 1 linje å legge inn, så tenker det er greit å legge den inn med engang nå som vi vet om den og hvor lett den er å oppdatere :)